### PR TITLE
Write Fee values to oracle

### PR DESCRIPTION
**Pull request by siosw**
Magi should update `l1FeeOverhead` and `l1FeeScalar` in the oracle

### CODE REVIEW

File: src/specular/sequencing/mod.rs
Comment (LEAFERx) [inline src/specular/sequencing/mod.rs]: `AttributesBuilder` holds the `config` so you can read directly

**Review by LEAFERx — CHANGES_REQUESTED** [review]
lets wait for #23 before merge

**Review by ujvl — APPROVED** [review]

**Review by LEAFERx — APPROVED** [review]

**Pull request merged**
